doc/source/overview/config_files.md

```diff
@@ -54,21 +54,17 @@ detector = load_detector(filepath)
 
 Compared to *standard instantiation*, config-driven instantiation has a number of advantages:
 
-- **Human readable**: The `config.toml` files are human-readable, providing a readily accessible record of 
-previously created detectors.
+- **Human readable**: The `config.toml` files are human-readable (and editable!), providing a readily accessible record 
+of previously created detectors.
 - **Flexible artefact specification**: Artefacts such as datasets and models can be specified as locally serialized
 objects, or as runtime registered objects (see [Specifying complex fields](complex_fields)). Multiple detectors can 
 share the same artefacts, and they can be easily swapped.
-- **Editable**: Config files written by the {func}`~alibi_detect.saving.save_detector` function can be edited 
-prior to reloading.
 - **Inbuilt validation**: The {func}`~alibi_detect.saving.load_detector` function uses
-[pydantic](https://pydantic-docs.helpmanual.io/) to validate detector configurations, improving reliability.
+[pydantic](https://pydantic-docs.helpmanual.io/) to validate detector configurations.
 
-In what follows, the Alibi Detect config files are explored in some detail. To get a general idea 
-of the expected layout of a config file, readers can also skip ahead to [Example config files](examples) for examples of
-config files for some common use cases. Alternatively, to obtain a fully populated config file for reference, users
-can run one of the [example notebooks](../cd/examples.md) and generate a config file by passing an instantiated 
-detector to {func}`~alibi_detect.saving.save_detector`.
+To get a general idea of the expected layout of a config file, see the [Example config files](examples). Alternatively, 
+to obtain a fully populated config file for reference, users can run one of the [example notebooks](../cd/examples.md) 
+and generate a config file by passing an instantiated detector to {func}`~alibi_detect.saving.save_detector`.
```

Comment:
Note that the change to link style hasn't been applied to these ones, unlike above.